tools/depends/native/perlmodule-parseyapp/Makefile

```diff
@@ -0,0 +1,45 @@
+include ../../Makefile.include
+PREFIX=$(NATIVEPREFIX)
+PLATFORM=$(NATIVEPLATFORM)
+DEPS= ../../Makefile.include Makefile
+
+# lib name, version
+LIBNAME=Parse-Yapp
+VERSION=1.21
+SOURCE=$(LIBNAME)-$(VERSION)
+ARCHIVE=$(SOURCE).tar.gz
+
+BASE_URL=https://www.cpan.org/authors/id/W/WB/WBRASWELL
```

Comment:
when jenkins is done, ive got this line pulled out into the temp BASE_URL commit i'll push. Will be removed once on mirrors as always